### ISSUE TITLE
about image size of training set

### ISSUE THREAD

**Issue by MahlerMozart** [warmspringwinds/pytorch-segmentation-detection#2]
Hello Daniil,
In the training process of your ResNet-8s, I notice that you crop all training images to 224x224 (RandomCropJoint(crop_size=(224, 224))). But you didn't adopt this approach when you train your FCN-32s model. Is it because the ResNet pretrained model is used as initial weights so we need to comply with its input image size (224x224) too? Do you think other input size can be used for training, without causing accuracy decline? Please advice. Thanks.

**Comment by warmspringwinds** [comment]
Hi, @MahlerMozart 
FCN-32s uses VGG-16 under the hood. In this case, the resnet is used.

Due to differences in models (more specifically, usage of Batch normalization
layers in resnet), we use different learning rate, batch size, crop size. This was done
more or less empirically.

Let me know if it is clear enough.

**Comment by MahlerMozart** [comment]
Hi Daniil,
That makes a lot of sense. I will try to train your model on my dataset to see how it works. Thanks.
